Rename Mac webpushd sandbox profile

bnham opened the pull request:
#### 18dc72d89c82e9201cc35a7e35c4667675fc8a04
<pre>
Rename Mac webpushd sandbox profile
<a href="https://bugs.webkit.org/show_bug.cgi?id=249448">https://bugs.webkit.org/show_bug.cgi?id=249448</a>
&lt;rdar://problem/103433185&gt;

Reviewed by Per Arne Vollan.

Because of the way the various iOS sandbox scripts are written, it&apos;s easiest if the iOS sandbox
profile has the name com.apple.WebKit.webpushd.sb and the Mac sandbox file has a different
name. This renames the Mac webpushd sandbox profile to com.apple.WebKit.webpushd.mac.sb.

* Source/WebKit/Configurations/WebKit.xcconfig:
* Source/WebKit/DerivedSources-input.xcfilelist:
* Source/WebKit/DerivedSources-output.xcfilelist:
* Source/WebKit/DerivedSources.make:
* Source/WebKit/PlatformMac.cmake:
* Source/WebKit/WebKit.xcodeproj/project.pbxproj:
* Source/WebKit/webpushd/WebPushDaemonMain.mm:
(WebKit::applySandbox):
* Source/WebKit/webpushd/mac/com.apple.WebKit.webpushd.mac.sb.in: Renamed from Source/WebKit/webpushd/mac/com.apple.WebKit.webpushd.sb.in.

Canonical link: <a href="https://commits.webkit.org/258018@main">https://commits.webkit.org/258018@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fbade9e7f4133566a91df95179afc2fc2f7dedf9

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/100665 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/9810 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/33706 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/109967 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/10745 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/85/builds/330 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/93066 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/107812 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/106446 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/8113 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/91373 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/34730 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/90030 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/22761 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/77700 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/3510 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/24291 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/3529 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/85/builds/330 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/9641 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/43788 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/5500 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/5316 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

beidson reviewed (approved):
From my perspective, this is an "if it builds. great!" patch

pvollan reviewed (approved):
R=me.

bnham committed:
Rename Mac webpushd sandbox profile
https://bugs.webkit.org/show_bug.cgi?id=249448
<rdar://problem/103433185>

Reviewed by Per Arne Vollan.

Because of the way the various iOS sandbox scripts are written, it's easiest if the iOS sandbox
profile has the name com.apple.WebKit.webpushd.sb and the Mac sandbox file has a different
name. This renames the Mac webpushd sandbox profile to com.apple.WebKit.webpushd.mac.sb.

* Source/WebKit/Configurations/WebKit.xcconfig:
* Source/WebKit/DerivedSources-input.xcfilelist:
* Source/WebKit/DerivedSources-output.xcfilelist:
* Source/WebKit/DerivedSources.make:
* Source/WebKit/PlatformMac.cmake:
* Source/WebKit/WebKit.xcodeproj/project.pbxproj:
* Source/WebKit/webpushd/WebPushDaemonMain.mm:
(WebKit::applySandbox):
* Source/WebKit/webpushd/mac/com.apple.WebKit.webpushd.mac.sb.in: Renamed from Source/WebKit/webpushd/mac/com.apple.WebKit.webpushd.sb.in.

Canonical link: https://commits.webkit.org/258018@main
